ctapipe/image/reducer.py

```diff
@@ -42,5 +112,55 @@ class NullDataVolumeReducer(DataVolumeReducer):
     Perform no data volume reduction
     """
 
-    def __call__(self, waveforms):
-        return waveforms
+    def select_pixels(self, waveforms, telid=None, selected_gain_channel=None):
+        mask = waveforms != 0
+        return mask
+
+
+class TailCutsDataVolumeReducer(DataVolumeReducer):
+    """
+    Reduce the time integrated shower image in 3 Steps:
+
+    1) Select pixels with tailcuts_clean.
+    2) Add iteratively all pixels with Signal S >= boundary_thresh
+       with ctapipe module dilate until no new pixels were added.
+    3) Adding new pixels with dilate to get more conservative.
+    """
+    end_dilates = IntTelescopeParameter(
+        default_value=1,
+        help="Number of how many times to dilate at the end."
+    ).tag(config=True)
+    iteration_steps = BoolTelescopeParameter(
```

Comment:
Iteration steps sound like an Int, to like a bool. 

Maybe call it `do_boundary_dilation`?